ObsCoreExtensionForRadioData.tex

```diff
@@ -42,7 +42,7 @@
 \begin{document}
 
 \begin{abstract}
-This is a proposed extension to the Obscore specification for description of radio data.
+This is a proposed extension to the ObsCore specification for discovery and selection of radio data.
```

Comment:
discovery a,d selction OK, but the "description" is an important concept. Should be reintroduced.